src/Files.Backend/Services/Settings/IFoldersSettingsService.cs

```diff
@@ -53,6 +78,31 @@ public interface IFoldersSettingsService : IBaseSettingsService, INotifyProperty
 		/// </summary>
 		FolderLayoutModes DefaultLayoutMode { get; set; }
 
+		/// <summary>
+		/// Gets or sets a value indicating git status column's default width
+		/// </summary>
+		double GitStatusColumnWidth { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating git last commit date column's default width
+		/// </summary>
+		double GitLastCommitDateColumnWidth { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating git last commit message column's default width
+		/// </summary>
+		double GitLastCommitMessageColumnWidth { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating git last commit author column's default width
+		/// </summary>
+		double GitCommitAuthorColumnWidth { get; set; }
```

Comment:
```suggestion
		double GitAuthorColumnWidth { get; set; }
```

Reply:
These are implied

Reply:
To be consistent with XAML, we should leave as it is.